unstructured_inference/inference/elements.py

```diff
@@ -30,6 +30,7 @@ class Rectangle:
     y1: Union[int, float]
     x2: Union[int, float]
     y2: Union[int, float]
+    source: Optional[str] = None
```

Comment:
This would make more sense in one of the child classes like `LayoutElement` or `TextRegion`.

Reply:
Could be, but ...isn't possible that some method creates just Rectangles?

Reply:
I think `TextRegion` would make more sense.

Reply:
Refactoring...